app/src/main/java/com/github/se/cyrcle/ui/map/MapScreen.kt

```diff
@@ -252,44 +285,33 @@ fun MapScreen(
           enabled = enableParkingAddition,
           colorLevel = ColorLevel.PRIMARY,
           testTag = "addButton")
+
+      IconButton(
+          icon = Icons.Default.MyLocation,
+          contentDescription = "Recenter on Location",
+          modifier =
+              Modifier.align(Alignment.BottomEnd)
+                  .padding(bottom = 25.dp, end = 16.dp)
+                  .scale(1.2f)
+                  .testTag("recenterButton"),
+          onClick = {
+            mapViewModel.updateTrackingMode(!mapViewModel.isTrackingModeEnable.value)
+
+            mapViewportState.transitionToFollowPuckState(
+                FollowPuckViewportStateOptions.Builder()
+                    .pitch(0.0)
+                    .zoom(maxZoom)
+                    .padding(EdgeInsets(500.0, 100.0, 100.0, 100.0))
```

Comment:
There is weird "glitch"/"shake" when we go off the focus mode. 
I think it's due to this Insets being 500 while all other one are 100.

I tried modifying it and it seems to resolve the shake

Reply:
Okay i will change that thanks !